packages/web/src/components/Learn/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { Icon, Link, Stack } from "@fluentui/react";
+import classNames from "classnames";
+import * as React from "react";
+
+import LearnDiagramBFF from "../../../assets/learn-diagram-bff.png";
+import LearnDiagramCSV from "../../../assets/learn-diagram-csv.png";
+import LearnDiagramViewer from "../../../assets/learn-diagram-viewer.png";
+
+import styles from "./Learn.module.css";
+
+// Public-facing learning/about page
+export default function Learn() {
+    return (
+        <div className={styles.root}>
+            <div className={styles.banner}>
+                <div className={styles.bannerBlur}>
+                    <div className={classNames(styles.section, styles.titleContainer)}>
+                        <h1>Curate your data your way</h1>
+                        <p>
+                            BioFile Finder helps users effortlessly interact with, extrapolate, and
```

Comment:
nit: 'effortlessly' is a very strong claim.  'Conveniently' or 'Easily'?  just a thought